### PR TITLE
docs(Scheduler): Add resource grouping header template documentation

### PR DESCRIPTION
**Pull request by NansiYancheva**
Related to https://github.com/telerik/blazor/issues/9886

### CODE REVIEW

File: components/scheduler/templates/resource-grouping-header.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+position: 13
+---
+
+# Resource Grouping Header Templates
+
+You can use the `SchedulerResourceGroupHeaderTemplate` to customize the rendering of the Scheduler resource grouping header cells. This allows you to change the appearance of the content, add custom content or any HTML elements.
+
+The `SchedulerResourceGroupHeaderTemplate` is invoked for each resource when the Scheduler is configured to have resources and grouping. The template applies in both horizontal and vertical grouping.
+
+The `SchedulerResourceGroupHeaderTemplate` can be defined at the root level of the Scheduler and individually for each [Scheduler views]({%slug scheduler-views-overview%}). When configured at the root, the template applies to all views. If a `SchedulerResourceGroupHeaderTemplate` is defined at the view level, it will override the root-level template for that specific view.
```
Comment (ntacheva) [inline components/scheduler/templates/resource-grouping-header.md:17]: I recommend listing the two declaration options in a bullet list for easier readability. The details for each declaration can be listed next to the respective bullet.

File: components/scheduler/templates/resource-grouping-header.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+The `context` of the template is a `SchedulerResourceGroupHeaderTemplateContext` object that contains:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `Text` | `string` | TThe group text. |
```
Comment (ntacheva) [inline components/scheduler/templates/resource-grouping-header.md:23]:
```suggestion
| `Text` | `string` | The group text. |
```

File: components/scheduler/templates/resource-grouping-header.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+| `Value` | `object` | The resource value.|
+| `Field` | `string` | The field of the resource. |
+
+>caption Example of using the SchedulerResourceGroupHeaderTemplate
+
+````CSHMTL
+<TelerikScheduler Data="@Appointments"
+                  @bind-Date="@StartDate"
+                  Height="600px">
```
Comment (ntacheva) [inline components/scheduler/templates/resource-grouping-header.md:32]:
```suggestion
                  Height="600px"
                  Width="900px">
```

**Review by ntacheva — APPROVED** [review]

**Review by ikoevska — COMMENTED** [review]
Ignored. Too old.

**Pull request merged**